daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -0,0 +1,120 @@
+---
+type: docs
+title: "Workflow API reference"
+linkTitle: "Workflow API"
+description: "Detailed documentation on the workflow API"
+weight: 900
+---
+## Component format
+
+A Dapr `workflow.yaml` component file has the following structure:
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+spec:
+  type: workflow.<TYPE>
+  version: v1.0-alpha1
+  metadata:
+  - name: <NAME>
+    value: <VALUE>
+ ```
+| Setting | Description |
+| ------- | ----------- |
+| `metadata.name` | The name of the workflow component. |
+| `spec/metadata` | Additional metadata parameters specified by workflow component |
+
+
+
+## Supported workflow methods
+
+### POST start workflow request
+```bash
+POST http://localhost:3500/v1.0-alpha1/workflows/{workflowComponent}/{workflowType}/{instanceId}/start
+```
```

Comment:
There needs to be examples here for Dapr and Temporal

If I use Dapr workflow with the HTTP API, I presume that I will have to specify the component type. Given this is in Dapr how do I use this API?
 But what is the component?
POST http://localhost:3500/v1.0-alpha1/workflows/{workflowComponent}/workflow.dapr/123/start